### PR TITLE
Adds ver command option to return generated version

### PR DESCRIPTION
**Pull request by chinghwayu**
Feel free to change the command from `ver` to something else.

```
$ miniver
usage: miniver [-h] [-v] {install,ver} ...

Interact with miniver

positional arguments:
  {install,ver}
    install      Install miniver into the current Python package
    ver          Get generated version

optional arguments:
  -h, --help     show this help message and exit
  -v, --version  show program's version number and exit
```
```
$ miniver ver .
0.8.0-dev2+g133c339
```

### CODE REVIEW

File: bin/miniver [hunk 1]
```diff
@@ -192,6 +201,15 @@ def install(args):
     print("\n".join((msg, _setup_template)).format(package_dir=package_dir))
 
 
+def ver(args):
+    package_dir = args.package_directory
+    version_location = glob.glob(f"{package_dir}/**/_version.py")[0]
```
Comment (jbweston) [inline bin/miniver:206]: Testing this out, it seems that the glob returns empty if `_version.py` is directly inside `package_dir`.

It seems that adding [`recursive=True`](https://docs.python.org/3/library/glob.html) will fix this.
We can also catch the cases when `_version.py` does not exist / more than 1 is found:

```suggestion
    try:
        version_location, = glob.glob(f"{package_dir}/**/_version.py", recursive=True)
    except ValueError as err:
        if "not enough" in str(err):
            print(f"'_version.py' not found in '{package_dir}'", file=sys.stderr)
            sys.exit(1)
        elif "too many" in str(err):
            print(f"More than 1 '_version.py' not found in '{package_dir}'",  file=sys.stderr)
            sys.exit(1)
        else:
            raise
```

**Review by jbweston — APPROVED** [review]
Thanks for the great addition @chinghwayu!

I think it would also be good to add tests to `.github/workflows/test.yml`. They can be essentially the same as the existing tests, except that we would invoke `miniver ver` directly rather than calling the Python interpreter.

I can also add this later if you don't have time.

Thanks again!

File: bin/miniver (hunk 1 above)
Comment (chinghwayu) [reply bin/miniver]: added

**Comment by chinghwayu** [comment]
> I think it would also be good to add tests to `.github/workflows/test.yml`. They can be essentially the same as the existing tests, except that we would invoke `miniver ver` directly rather than calling the Python interpreter.

Writing tests seem easy enough but I don't really work with Github actions and don't have an environment setup to run act.

I'll leave this to you...

**Pull request merged**